hack/make-rules/test.sh

```diff
@@ -263,6 +265,7 @@ runTests() {
           ${testargs[@]:+${testargs[@]}} \
         | tee ${junit_filename_prefix:+\"${junit_filename_prefix}-\$_pkg_out.stdout\"} \
         | grep \"${go_test_grep_pattern}\"" \
+          {} \
```

Comment:
Sure, do you mean just to align it with the `|` character on the previous two lines?